sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetColumnIndexSuite.scala

```diff
@@ -31,24 +31,25 @@ class ParquetColumnIndexSuite extends QueryTest with ParquetTest with SharedSpar
    *  |-------|-----|-----|---|---|---|---|---|
    * col_2   400   300   200 200 200 200 200 200
```

Comment:
BTW, about the layout, the total of col_1 or col_2 is all 2000? But looks like col_2 has only 1900?

Reply:
opps you are right - it should be 400, 300, 300, 200, 200, 200, 200, 200

Reply:
And also the PR description. :)